Spellcheck fails if no changed src files

### 🐛 Describe the bug

In case there are no changes to a markdown file, the spellcheck is run with [empty arguments](https://github.com/pytorch/serve/blob/master/ts_scripts/spellcheck.sh#L15) which throws the [error](https://github.com/marketplace/actions/github-spellcheck-action-german#diagnostic-text-runtimeerror-none-of-the-source-targets-from-the-configuration-match-any-files) `RuntimeError: None of the source targets from the configuration match any files:`

### Error logs

e.g. https://github.com/pytorch/serve/actions/runs/3291314541/jobs/5425303921

```
sources=""
  for file in ; do
    sources+=" -S $file"
  done
  pyspelling -c $GITHUB_WORKSPACE/ts_scripts/spellcheck_conf/spellcheck.yaml --name Markdown $sources
  shell: /usr/bin/bash -e {0}
Traceback (most recent call last):
  File "/home/runner/.local/bin/pyspelling", line 8, in <module>
    sys.exit(main())
  File "/home/runner/.local/lib/python3.8/site-packages/pyspelling/__main__.py", line 30, in main
    return run(
  File "/home/runner/.local/lib/python3.8/site-packages/pyspelling/__main__.py", line 55, in run
    for results in spellcheck(
  File "/home/runner/.local/lib/python3.8/site-packages/pyspelling/__init__.py", line 673, in spellcheck
    for result in spellchecker.run_task(task, source_patterns=sources):
  File "/home/runner/.local/lib/python3.8/site-packages/pyspelling/__init__.py", line 317, in run_task
    for sources in self._walk_src(source_patterns, glob_flags, glob_limit, self.pipeline_steps, expect_match):
  File "/home/runner/.local/lib/python3.8/site-packages/pyspelling/__init__.py", line 213, in _walk_src
    raise RuntimeError(
RuntimeError: None of the source targets from the configuration match any files:

Error: Process completed with exit code 1.
```

### Installation instructions

Failure in CI logs

### Model Packaing

N/A

### config.properties

N/A

### Versions

CI build from current master

### Repro instructions

Run CI on any PR without changed source files e.g. https://github.com/pytorch/serve/actions/runs/3291314541/jobs/5425303921

### Possible Solution

Skip if [sources](https://github.com/pytorch/serve/blob/master/ts_scripts/spellcheck.sh#L5-L9) is empty

--- a/ts_scripts/spellcheck.sh
+++ b/ts_scripts/spellcheck.sh
@@ -9,7 +9,12 @@ else
 fi
 
 sources_arg=""
-for src in $sources ;do
-        sources_arg+=" -S $src"
+for src in $sources; do
+        sources_arg="${sources_arg} -S $src"
 done
-pyspelling -c ts_scripts/spellcheck_conf/spellcheck.yaml --name Markdown $sources_arg
+
+if [ ! "$sources_arg" ]; then
+	echo "No files to spellcheck"
+else
+	pyspelling -c ts_scripts/spellcheck_conf/spellcheck.yaml --name Markdown $sources_arg
+fi

